inst/maintenance/unit-cpp/test-evtools.R

```diff
@@ -103,3 +113,28 @@ test_that("evtools - give timed dose now", {
   b <- mrgsim(mod, param = list(mode = 1))
   expect_identical(as.data.frame(a), as.data.frame(b))
 })
+
+test_that("evtools - replace", {
+  mod <- init(mod, C = 100)
+  mod <- update(mod, delta = 0.1)
+  a <- mrgsim(mod, param = list(mode = 7))
+  before <- filter(a, time < 5)
+  expect_true(all(before$C==100))
+  after <- filter(a, time >= 5)
+  expect_true(all(after$C==50))
+  
+  #' When the replacement is timed into the future, we see the 
```

Comment:
extreme nit-pick: I think it'd be better to use plain comments rather than roxygen-style ones. (I've been allergic to unnecessary roxygen-style comments ever since I spent time debugging [this issue](https://github.com/rstudio/rstudio/issues/12772).)

Reply:
Thanks, @kyleam; I reverted the comments and will try to stick to plain R comments going forward (I _think_ I do most of the time, but I'm sure there's tons of `#'` comments in the code too.)